server/src/test/java/org/apache/druid/client/ImmutableDruidDataSourceTest.java

```diff
@@ -26,39 +26,76 @@
 import com.google.common.collect.ImmutableSortedMap;
 import org.apache.druid.jackson.DefaultObjectMapper;
 import org.apache.druid.java.util.common.Intervals;
+import org.apache.druid.test.utils.TestUtils;
 import org.apache.druid.timeline.DataSegment;
 import org.apache.druid.timeline.DataSegment.PruneLoadSpecHolder;
-import org.junit.Assert;
 import org.junit.Test;
 
+import javax.annotation.Nonnull;
```

Comment:
You don't have to use this. We assume everything is not null unless they are explicitly annotated as `@Nullable`.

Reply:
Done